### PR TITLE
Add support for Endpoint DLP for MacOS

### PR DESCRIPTION
**Pull request by bryan-lipinski**
Updates to support the deployment of Endpoint DLP for MacOS as part of Microsoft Endpoint Technology:

- Profiles
  - New accessibility.mobileconfig for Endpoint DLP
  - Update to fulldisk.mobileconfig to include Endpoint DLP
- Schema 
  - Update preferences schema to include new DLP related settings
- Example Settings
  - Profile and schema for Endpoint DLP only deployments (No MDE) 
  - Profile for Microsoft AutoUpdate

### CODE REVIEW

File: macos/settings/data_loss_prevention/schema.json [hunk 1]
```diff
@@ -0,0 +1,434 @@
+{
+    "__feedback": "jmanifest@microsoft.com",
+    "__version": "101.13.75",
```
Comment (maximvelichko) [inline macos/settings/data_loss_prevention/schema.json:3]: That's the "from" version where this schema is supported. I try to use the _rough_ versions of wdav.pkg, starting with the one this schema is supported.

File: macos/settings/microsoft_auto_update/com.microsoft.autoupdate2.mobileconfig [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+            <key>EnableCheckForUpdatesButton</key>
+            <true/>
+            <key>DisableInsiderCheckbox</key>
+            <false/>
+            <key>SendAllTelemetryEnabled</key>
+            <true/>
+            </dict>
+        </array>
+    </dict>
+</plist>
```
Comment (maximvelichko) [inline macos/settings/microsoft_auto_update/com.microsoft.autoupdate2.mobileconfig:57]: Github warns "no new line at the end of file". I don't know how critical is that.

File: macos/schema/schema.json [hunk 3]
```diff
@@ -394,6 +394,41 @@
                     }
                 }
             }
```
Comment (maximvelichko) [inline macos/schema/schema.json:396]: See my comment for another schema.json (update the version number)

File: macos/settings/data_loss_prevention/schema.json (hunk 1 above)
Comment (bryan-lipinski) [reply macos/settings/data_loss_prevention/schema.json]: Updated to current release: 2108-2

File: macos/settings/microsoft_auto_update/com.microsoft.autoupdate2.mobileconfig (hunk 2 above)
Comment (bryan-lipinski) [reply macos/settings/microsoft_auto_update/com.microsoft.autoupdate2.mobileconfig]: Fixed.  (and the other plist)

File: macos/settings/data_loss_prevention/schema.json (hunk 1 above)
Comment (bryan-lipinski) [reply macos/settings/data_loss_prevention/schema.json]: Updated to the current release: 2108-2

File: macos/settings/microsoft_auto_update/com.microsoft.autoupdate2.mobileconfig (hunk 2 above)
Comment (bryan-lipinski) [reply macos/settings/microsoft_auto_update/com.microsoft.autoupdate2.mobileconfig]: Added a new line to this file and com.microsoft.wdav.plist

**Review by maximvelichko — APPROVED** [review]

**Pull request merged**